src/_cffi_src/openssl/ssl.py

```diff
@@ -470,17 +470,19 @@
     return 1;
 }
 #endif
+
 /* Added in 1.0.2 but we need it in all versions now due to the great
    opaquing. */
-#if OPENSSL_VERSION_NUMBER < 0x10002001L || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Ok; is it just this specific line or are there others as well?

On Tue, Jul 5, 2016 at 12:46 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/_cffi_src/openssl/ssl.py
> <https://github.com/pyca/cryptography/pull/3037#discussion_r69597499>:
>
> >  /* Added in 1.0.2 but we need it in all versions now due to the great
> >     opaquing. */
> > -#if OPENSSL_VERSION_NUMBER < 0x10002001L || defined(LIBRESSL_VERSION_NUMBER)
>
> I don't remember specifically, but I know we had to change a definition
> much like this one to avoid a bug last year. You can probably dig it up if
> you hunt through the history, but that's a lot of work.
>
> I'm extremely reluctant to merge a change like this though because I'm
> almost certain it will cause breakage as is.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pyca/cryptography/pull/3037/files/d2bca036cbc79f1be53c92886048d4f267a7a13f#r69597499>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAADBC-5Usrb_FWA3b32Nei-vAYMZ-Hdks5qSoppgaJpZM4JDuXx>
> .
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6